dotcom-rendering/src/footballMatches.test.ts

```diff
@@ -197,4 +198,25 @@ describe('footballMatches', () => {
 			expect(match.homeTeam.name).toBe(cleanName);
 		}
 	});
+	it('should replace known live match status with our status', () => {
+		const matchDay = okOrThrow(
+			parseLiveMatch(matchDayLiveSecondHalf),
+			'Expected football live match parsing to succeed',
+		);
+
+		expect(matchDay.status).toBe('2nd');
```

Comment:
Do you need to export `parseLiveMatch` for the tests? `parse` should end up calling this function, so you can do:

```suggestion
		const matchDay = okOrThrow(
			parse(withMatches([matchDayLiveSecondHalf]))
			'Expected football live match parsing to succeed',
		);
		
		const match = matchDay[0]!.competitions[0]!.matches[0];

		if (match?.kind !== 'Live') {
			throw new Error('Expected live match');
		}

		expect(match.status).toBe('2nd');
```

This keeps the public API for this module to only what's needed by production code, and allows us to refactor its internals without updating the tests as long as that API doesn't change.